ext/pdo/pdo_stmt.stub.php

```diff
@@ -5,10 +5,10 @@
 class PDOStatement implements IteratorAggregate
 {
     /** @return bool */
-    public function bindColumn(string|int $column, mixed &$param, int $type = 0, int $max_length = 0, mixed $driver_options = null) {}
+    public function bindColumn(string|int $column, mixed &$var, int $type = 0, int $max_length = 0, mixed $driverOptions = null) {}
 
     /** @return bool */
-    public function bindParam(string|int $param, mixed &$bind_var, int $type = PDO::PARAM_STR, int $max_length = 0, mixed $driver_options = null) {}
+    public function bindParam(string|int $param, mixed &$var, int $type = PDO::PARAM_STR, int $max_length = 0, mixed $driverOptions = null) {}
```

Comment:
You changed `$driverOptions` to `$options` in `PDO::prepate()`. is there any difference between the two use-cases?

(there is one one `$max_length` here)

Reply:
For PDO::prepare(), the documentation says this:

> This array holds one or more key=>value pairs to set attribute values for the PDOStatement object that this method returns. You would most commonly use this to set the PDO::ATTR_CURSOR value to PDO::CURSOR_SCROLL to request a scrollable cursor. Some drivers have driver-specific options that may be set at prepare-time.

So it seems that at least some of the supported options are generic PDO options, not driver-specific ones.

For `$driverOptions` on these APIs, it seems like the parameter is effectively unused. This sets the driver_params fields, but it is never actually used anywhere.